stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -122,12 +122,29 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
-          - name: RABBITMQ_USE_LONGNAME
-            value: "true"
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: K8S_SERVICE_NAME
+            value: "{{ template "rabbitmq.fullname" . }}-headless"
+          - name: K8S_ADDRESS_TYPE
+            value: {{ default "ip" .Values.rabbitmq.clustering.address_type }}
```

Comment:
using a `default` value in the template is not recommended. The default value should come from the `values.yaml` file